web3.js/test/connection.test.ts

```diff
@@ -1167,6 +1167,44 @@ describe('Connection', function () {
       });
 
       describe('block height strategy', () => {
+        it('rejects if called with an already-aborted `abortSignal`', () => {
```

Comment:
Side note: I would have written more test coverage here, to assert that when cancelled, the subscription gets disposed and the nonce account no longer gets pinged. Our test framework, however, _just doesn't work_. I tried setting up `sinon` spies, and they never fired. I tested _other_ spies in previous tests and – surprise – they don't work either.

I, for one, can't wait to return to Jest as we rewrite web3.js.